metadata/src/main/java/org/apache/kafka/image/loader/MetadataBatchLoader.java

```diff
@@ -152,7 +153,8 @@ public long loadBatch(Batch<ApiMessageAndVersion> batch, LeaderAndEpoch leaderAn
                     .numBytes(numBytes)     // This will be zero if we have not yet read a batch
                     .build();
                 if (log.isDebugEnabled()) {
-                    log.debug("handleCommit: Generated a metadata delta between {} and {} from {} batch(es) in {} us.",
+                    log.debug("handleCommit: Generated a {} metadata delta between {} and {} from {} batch(es) in {} us.",
+                            provenance.isOffsetBatchAligned() ? "batch-aligned" : "non-batch-aligned",
```

Comment:
Now that I see this, let's move the "aligned" part to the end as a separate phrase. 

E.g., 

```
handleCommit: Generated a metadata delta between 0 and 10 from 2 batch(es) in 4 us. The delta is batch aligned
```